### PR TITLE
Update getExternalCodeSize, bytecode and bytecode size is different

### LINKED ISSUE #60: Change getExternalCodeSize to not rely on wasm compiler
`getExternalCodeSize` checks for an account of `code: (module)`. I would suggest to replace that with the actual output code, so this test would not need to be updated if bytecode size changes.

### PR DESCRIPTION
**Pull request by hugo-dc**
Fixes #60.

### CODE REVIEW

File: src/GeneralStateTestsFiller/stEWASMTests/getExternalCodeSizeFiller.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ getExternalCodeSize:
           (export "main" (func $main))
 
           (func $main
-            ;; for now just use our own address since it's easier
+            ;; check this account codesize
```
Comment (axic) [inline src/GeneralStateTestsFiller/stEWASMTests/getExternalCodeSizeFiller.yml]: I think this still uses the `0xdeadbeef` account as that's the one in the data section.

**Comment by axic** [comment]
Please replace `code: (module)` with a fixed wasm bytecode and use that as a reference.

File: src/GeneralStateTestsFiller/stEWASMTests/getExternalCodeSizeFiller.yml (hunk 1 above)
Comment (hugo-dc) [reply src/GeneralStateTestsFiller/stEWASMTests/getExternalCodeSizeFiller.yml]: Yes it is still using `0xdeadbeef` here https://github.com/ewasm/tests/blob/b856f9eb2c9589f703e5bc1e7b9376f2faaaa203/src/GeneralStateTestsFiller/stEWASMTests/getExternalCodeSizeFiller.yml#L43-L48 , but it also check its own code size here: https://github.com/ewasm/tests/blob/b856f9eb2c9589f703e5bc1e7b9376f2faaaa203/src/GeneralStateTestsFiller/stEWASMTests/getExternalCodeSizeFiller.yml#L38-L41

**Comment by hugo-dc** [comment]
@axic I will replace `code: (module)` with the corresponding wasm bytecode, but the main account is also checking its own bytecode length, should I avoid this and just check the external account bytecode?

**Comment by axic** [comment]
Yeah I think it is not necessary to check do `getaddress`+`getexternalbalance` as long as there is another test case which checks the validity of `getaddress`.

**Review by lrettig — APPROVED** [review]
The test passes for me and the filler looks good

**Comment by hugo-dc** [comment]
@axic @lrettig I made a change in order to add the wasm bytecode instead the wast code in the external account, I also made a change in order to avoid checking the account's own code size. 

There is something interesting about this change, in previous test case I didn't consider transaction cost (21000) and the test was passing anyway, for this new change I needed to add the transaction cost to the calculation. I will look into this.

**Review by axic — APPROVED** [review]

**Pull request merged**